src/main/com/fulcrologic/rad/rendering/semantic_ui/semantic_ui_controls.cljc

```diff
@@ -48,6 +49,13 @@
               :pick-many entity-picker/to-many-picker}}
 
    ;; Report-related controls
+   :com.fulcrologic.rad.report/type->formatter
+   {:string  (fn [report-instance value] value)
+    :instant (fn [report-instance value] (formatters/inst->human-readable-date value))
+    :int     (fn [report-instance value] (formatters/numeric->str value))
+    :decimal (fn [report-instance value] (formatters/numeric->str value))
+    :boolean (fn [report-instance value] (str value))}
```

Comment:
Boolean should probably be an if, so nils print right `(if value "true" "false")`